stores/github/authorization-model.json

```diff
@@ -151,6 +216,69 @@
         "repo_writer": {
           "this": {}
         }
+      },
+      "metadata": {
+        "relations": {
+          "member": {
+            "directly_related_user_types": [
+              {
+                "type": "organization"
```

Comment:
I am confused - shouldn't organization member be user?

Reply:
Fixed